crates/api/src/constraints/api.rs

```diff
@@ -25,6 +26,22 @@ where
     auctioneer: Arc<A>,
     db: Arc<DB>,
     chain_info: Arc<ChainInfo>,
+
+    constraints_handle: ConstraintsHandle,
+}
+
+// TODO: Move this to appropriate place
+#[derive(Clone)]
+pub struct ConstraintsHandle {
+    pub(crate) constraints_tx: mpsc::Sender<ConstraintsMessage>, 
+}
```

Comment:
Where would you like to put this?

Reply:
On second thought I think this might be right place itself. lmk if any suggestions